### PR TITLE
MLPAB-732: Add what happens when you sign page

### PR DESCRIPTION
**Pull request by acsauk**
# Purpose

I got tired of printing `Lpa{}` props in templates to check if I'd set cypress tests up right so I've added a tmplfn for easier dumping of structs in templates. 

Fixes MLPAB-732

### CODE REVIEW

File: app/internal/page/data.go [hunk 1]
```diff
@@ -364,3 +364,8 @@ func ChooseReplacementAttorneysState(lpa *Lpa) TaskState {
 
 	return TaskCompleted
 }
+
+// TODO update references in code from hw to pw and pfa to pa
```
[the remark this answers is not in the document]
Comment (hawx) [reply app/internal/page/data.go:368]: also it is much easier to see the difference between hw/pfa than pa/pf

**Review by hawx — DISMISSED** [review]

File: app/internal/page/data.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (acsauk) [reply app/internal/page/data.go:368]: Fair point - I'll switch this to `IsHealthAndWelfareLpa()`

File: web/template/attorney_what_happens_when_you_sign.gohtml [hunk 2]
```diff
@@ -7,9 +7,11 @@
         <div class="govuk-grid-column-two-thirds">
             <h1 class="govuk-heading-xl">{{ tr .App "whatHappensWhenYouSignTheLpaTitle" }}</h1>
 
+            {{ printf "%#v\n" .Lpa }}
```
Comment (hawx) [inline web/template/attorney_what_happens_when_you_sign.gohtml:10]: you'll want to remove this

Comment (acsauk) [reply web/template/attorney_what_happens_when_you_sign.gohtml]: Ugh - thanks, dropped now 👍

File: app/internal/page/attorney/register.go [hunk 3]
```diff
@@ -111,8 +111,10 @@ func Register(
 		YourAddress(logger, tmpls.Get("your_address.gohtml"), addressClient, lpaStore))
 	handleRoot(page.Paths.Attorney.ReadTheLpa, RequireLpa,
 		ReadTheLpa(tmpls.Get("attorney_read_the_lpa.gohtml"), lpaStore))
-	handleRoot(page.Paths.Attorney.RightsAndResponsibilities, RequireSession,
+	handleRoot(page.Paths.Attorney.RightsAndResponsibilities, None,
```
Comment (hawx) [inline app/internal/page/attorney/register.go:114]: I think this should `RequireSession`, or the continue button won't work. Also we already have a general rights and responsibilities page linked from the footer that sits outside the flows.

**Pull request merged**